youboraplugin/src/main/java/com/kaltura/playkit/plugins/youbora/YouboraPlugin.java

```diff
@@ -274,6 +285,7 @@ private void handleTracksAvailable(PlayerEvent.TracksAvailable event, NPAWPlugin
     private static Options parseConfig(Object config) {
         if (config instanceof YouboraConfig) {
             houseHoldId = ((YouboraConfig) config).getHouseHoldId();
+            fastDataConfig = ((YouboraConfig) config).getFastDataConfig();
```

Comment:
@ealfonso93 
why is it used only here?
it will be null in 99%?

Reply:
usually jsonObject or Options are the common cases

Reply:
I'm not 100% sure but this one doesn't come from the options itself (like most of the parameters do), since we need this info (fastDataConfig) at the same time we set the options, but if you think it's better to put it on the jsonObject or Bundle it's doable

Reply:
@ealfonso93
please contact me in skype